modules/data.remote/inst/scripts/Download_OSU_AGB_data_v5.R

```diff
@@ -67,13 +67,13 @@ doParallel::registerDoParallel(cl)
 # get median data
 med_urls <- paste(URL,dataset_version,target_dataset,"median",med_files,sep="/")
 PEcAn.logger::logger.info("*** Downloading AGB median data")
-foreach::foreach(i=1:length(med_urls)) %dopar% try(download.file(med_urls[i], file.path(target_directory,med_files[i])))
+foreach::foreach(i=1:length(med_urls)) %dopar% try(PEcAn.utils::download_file(med_urls[i], file.path(target_directory,med_files[i])))
```

Comment:
This call won't fail since the argument is unnamed, but please keep using the base version (i.e. `utils::download.file`) so that this PR isn't changing which function existing calls use.